src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1616,6 +1616,22 @@ def getAcquisitionEra(self):
         """
         return getattr(self.data.parameters, 'acquisitionEra', None)
 
+    def setCampaignName(self, campaign):
+        """
+        _setTaskType_
+
+        Set the campaign name of this task
+        """
+        self.data.campaignName = campaign
+
+    def getCampaignName(self):
+        """
+        _getCampaignName_
```

Comment:
Please update the docstring according to the guidelines: https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices